src/static/css/homepage.css

```diff
@@ -0,0 +1,26 @@
+.asd {
+    padding: 60px;
+    text-align: center;
+    background: #6C7A89;
+    color: white;
+}  
+
+.asd > .title {
+    font-size: 50px;
+}
+.asd > .caption {
+    font-size: 20px;
+}
+
+.social-icons-container{
+    position:absolute;
+    top:0;
+    right:0; 
+    padding: 20px 10px;
+}
+
+.social-icons-container img{
+    width: 30px;
+    height: auto;  
+    padding-right: 10px
+}   
```

Comment:
trailing newline for the end of the file
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
I always wondered why some editors/linters added an extra line... TIL and changed!

Reply:
If you want to enforce it by linting, add 
```
    "eol-last": ["error", "always"],
```
to the `.eslintrc`!